src/api.js

```diff
@@ -488,12 +488,13 @@ class VLWholeSlideMicroscopyImageViewer {
         } else {
           // TODO: support "image/jp2" and "image/jls"
           const mimeType = 'image/jpeg';
+
           const retrieveOptions = {
             studyInstanceUID,
             seriesInstanceUID,
             sopInstanceUID,
             frameNumbers,
-            mimeType
+            mimeType: `${mimeType}; transfer-syntax=1.2.840.10008.1.2.4.50`
```

Comment:
I think that is fine, since there is no other JPEG transfer syntax that is used for whole-slide imaging.